src/server/implementation/mutations/userOAuth.ts

```diff
@@ -49,7 +48,7 @@ export async function userOAuthImpl(
     config,
   );
 
-  const code = generateRandomString(8, alphabet("0-9"));
+  const code = generateRandomString(8, "0123456789");
```

Comment:
`@oslojs/crypto` doesn’t have an equivalent `alphabet` function[^1] so I just hardcoded the expected values.

[^1]: See for instance the examples in their documentation: https://crypto.oslojs.dev/examples/random-values